locales/tr-TR/translation.json

```diff
@@ -0,0 +1,280 @@
+  "To address": "Adrese",
+  "Select the address to sweep the funds to.": "Paraların hangi adrese sürükleneceğini seçin.",
+  "sweepOperationFromTo": "Bu işlem tüm paraları <1>{{ from }}</1> ' den sürükleyip <3>{{ to }}</3> 'e transfer edecek.",
+  "Consolidate UTXOs": "UTXO (işlemleri) birleştir",
+  "Fee": "Ücret",
+  "Consolidate": "Birleştir",
+  "The group number will be automatically be appended to the addresses’ label.": "Grup numarası otomatik olarak adreslerin etiketlerine eklenecektir.",
+  "Advanced options": "Gelişmiş seçenekler",
+  "Group": "Grup",
+  "Select group": "Grubu seçin",
+  "Generate": "Üret",
+  "Secret recovery phrase": "Gizli kurtarma sözcükleri",
+  "Type your password to show the phrase.": "Sözcükleri görmek için şifrenizi yazın.",
+  "Show": "Göster",
+  "Carefully note down the words! They are your wallet's secret recovery phrase.": "Sözcükleri önemle not ediniz! Cüzdanınızın gizli kurtarma sözcükleridir.",
+  "No recovery phrase was stored along with this wallet": "Bu cüzdan için kurtarma sözcükleri kaydedilmedi",
+  "Expected fee": "Beklenen ücret",
+  "Address format is incorrect": "Adres biçimi yanlış",
+  "Select the address to send funds from.": "Paranın gönderileceği adresi seçin.",
+  "Recipient's address": "Alıcının adresi",
+  "Advanced settings": "Gelişmiş ayarlar",
+  "Gas amount": "Gaz miktarı",
+  "Gas price": "Gaz fiyatı",
+  "Check": "Kontrol et",
+  "Send": "Gönder",
+  "Password Check": "Şifre kontrolü",
+  "Error while building the transaction": "İşlem oluşturulurken hata oluştu",
+  "Transactions sent!": "İşlemler gönderildi!",
+  "Transaction sent!": "İşlem gönderildi!",
+  "Error while sending the transaction": "İşlem gönderilirken hata oluştu",
+  "Enter your password to send the transaction.": "İşlemi göndermek için şifrenizi girin.",
+  "Lock time": "Kilit süresi",
+  "Duration in minutes after which an idle wallet will lock automatically.": "Boşta beklenirken kaç dakika içinde adresin otomatik olarak kilitleneceği süreç.",
+  "Minutes": "Dakika",
+  "Off": "Kapalı",
+  "Theme": "Tema",
+  "Select the theme and please your eyes.": "Göz zevkinize uygun olan temayı seçin.",
+  "Toggle discreet mode (hide amounts).": "Gizli moda geç (miktarları gizle).",
+  "Password requirement": "Şifre gerekli",
+  "Require password confirmation before sending each transaction.": "Her işlemin gönderilmeden önce şifre doğrulaması gereklidir.",
+  "Password": "Şifre",
+  "Type your password to change this setting.": "Bu ayarı değiştirmek için şifrenizi yazın.",
+  "Enter": "Giriş",
+  "Custom network settings saved.": "Uyarlanmış ağ ayarları kaydedildi.",
+  "Make sure to always check what is the selected network before sending transactions.": "İşlemleri göndermeden önce hangi ağda olduğunuzdan emin olmak için her zaman kontrol edin.",
+  "Network": "Ağ",
+  "Node host": "Düğüm noktası barındırma hizmeti",
+  "Explorer API host": "Tarayıcı API(Upa) barındırma",
+  "Explorer URL": "Tarayıcı URL(adresi)",
```

Comment:
Is it a common thing in Turkish to add things in parentheses? :/

Also, does it make sense that there is no space before the parentheses?

<img width="578" alt="image" src="https://user-images.githubusercontent.com/1579899/218669166-eab6d2a6-c473-49e8-8df5-7da5a16fac73.png">